.vscode/settings.json

```diff
@@ -8,4 +8,7 @@
   "[typescriptreact]": {
     "editor.defaultFormatter": "esbenp.prettier-vscode"
   },
+  "[typescript]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  },
```

Comment:
files keep registering as `typescript`, not `typescriptreact`

is it fine we add `typescript` too?

Reply:
Sure. I haven't run into any issues myself with the current configuration, but I don't think it can hurt.